### PR TITLE
Add TiUP command docs to TOC

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

The links of TiUP command docs are missing in TOC. This PR adds these links.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM
Great work!

**Review by qqqdan — APPROVED** [review]

File: TOC.md [hunk 1]
```diff
@@ -183,8 +183,82 @@
...
+      - [命令概览](/tiup/tiup-reference.md)
+      - TiUP 命令
+        - [tiup clean](/tiup/tiup-command-clean.md)
+        - [tiup completion](/tiup/tiup-command-completion.md)
+        - [tiup env](/tiup/tiup-command-env.md)
+        - [tiup help](/tiup/tiup-command-help.md)
+        - [tiup install](/tiup/tiup-command-install.md)
+        - [tiup list](/tiup/tiup-command-list.md)
+        - tiup mirror
+          - [tiup mirror 命令概览](/tiup/tiup-command-mirror.md)
```
Comment (shichun-0415) [inline TOC.md:196]:
```suggestion
        - [tiup mirror 命令](/tiup/tiup-command-mirror.md)
```
概览跟下面的具体命令并列不太合适。

File: TOC.md [hunk 2]
```diff
@@ -183,8 +183,82 @@
...
+          - [tiup mirror clone](/tiup/tiup-command-mirror-clone.md)
+          - [tiup mirror genkey](/tiup/tiup-command-mirror-genkey.md)
+          - [tiup mirror grant](/tiup/tiup-command-mirror-grant.md)
+          - [tiup mirror init](/tiup/tiup-command-mirror-init.md)
+          - [tiup mirror merge](/tiup/tiup-command-mirror-merge.md)
+          - [tiup mirror modify](/tiup/tiup-command-mirror-modify.md)
+          - [tiup mirror publish](/tiup/tiup-command-mirror-publish.md)
+          - [tiup mirror rotate](/tiup/tiup-command-mirror-rotate.md)
+          - [tiup mirror set](/tiup/tiup-command-mirror-set.md)
+          - [tiup mirror sign](/tiup/tiup-command-mirror-sign.md)
```
Comment (shichun-0415) [inline TOC.md:206]: 感觉按这些命令实际使用顺序来排序会更合理些？下图是概览文件对命令的解释：
image

File: TOC.md [hunk 3]
```diff
@@ -183,8 +183,82 @@
     - [TiUP 组件管理](/tiup/tiup-component-management.md)
     - [FAQ](/tiup/tiup-faq.md)
     - [故障排查](/tiup/tiup-troubleshooting-guide.md)
-    - [参考手册](/tiup/tiup-reference.md)
+    - TiUP 命令参考手册
+      - [命令概览](/tiup/tiup-reference.md)
+      - TiUP 命令
```
Comment (shichun-0415) [inline TOC.md:188]: 下面的具体命令排序是按照什么规则呢？感觉应该按照实际使用时的命令顺序来排，比如 安装→查看状态→升级→......。第一个是 clean（清理组件数据）好像不太合适。

File: TOC.md [hunk 4]
```diff
@@ -183,8 +183,82 @@
...
+          - [tiup mirror publish](/tiup/tiup-command-mirror-publish.md)
+          - [tiup mirror rotate](/tiup/tiup-command-mirror-rotate.md)
+          - [tiup mirror set](/tiup/tiup-command-mirror-set.md)
+          - [tiup mirror sign](/tiup/tiup-command-mirror-sign.md)
+        - [tiup status](/tiup/tiup-command-status.md)
+        - [tiup telemetry](/tiup/tiup-command-telemetry.md)
+        - [tiup uninstall](/tiup/tiup-command-uninstall.md)
+        - [tiup update](/tiup/tiup-command-update.md)
+      - TiUP Cluster 命令
+        - [TiUP Cluster 命令概览](/tiup/tiup-component-cluster.md)
```
Comment (shichun-0415) [inline TOC.md:212]:
```suggestion
      - [TiUP Cluster 命令](/tiup/tiup-component-cluster.md)
```
下面具体的命令排序，看看要不要也根据实际使用命令的顺序排一下。

File: TOC.md [hunk 5]
```diff
@@ -183,8 +183,82 @@
...
+        - [tiup cluster replay](/tiup/tiup-component-cluster-replay.md)
+        - [tiup cluster restart](/tiup/tiup-component-cluster-restart.md)
+        - [tiup cluster scale-in](/tiup/tiup-component-cluster-scale-in.md)
+        - [tiup cluster scale-out](/tiup/tiup-component-cluster-scale-out.md)
+        - [tiup cluster start](/tiup/tiup-component-cluster-start.md)
+        - [tiup cluster stop](/tiup/tiup-component-cluster-stop.md)
+        - [tiup cluster template](/tiup/tiup-component-cluster-template.md)
+        - [tiup cluster upgrade](/tiup/tiup-component-cluster-upgrade.md)
+      - TiUP DM 命令
+        - [TiUP DM 命令概览](/tiup/tiup-component-dm.md)
```
Comment (shichun-0415) [inline TOC.md:238]:
```suggestion
      - [TiUP DM 命令](/tiup/tiup-component-dm.md)
```
具体 dm 命令的排序，修改建议同上。

File: TOC.md (hunk 3 above)
Comment (qiancai) [reply TOC.md:188]: 因为是参考手册，是字母顺序，方便检索。

File: TOC.md [hunk 6]
```diff
@@ -183,8 +183,79 @@
...
+    - TiUP 命令参考手册
+      - [命令概览](/tiup/tiup-reference.md)
+      - TiUP 命令
+        - [tiup clean](/tiup/tiup-command-clean.md)
+        - [tiup completion](/tiup/tiup-command-completion.md)
+        - [tiup env](/tiup/tiup-command-env.md)
+        - [tiup help](/tiup/tiup-command-help.md)
+        - [tiup install](/tiup/tiup-command-install.md)
+        - [tiup list](/tiup/tiup-command-list.md)
+        - [tiup mirror 命令](/tiup/tiup-command-mirror.md)
```
Comment (qiancai) [inline TOC.md:195]:
```suggestion
        - tiup mirror
          - [概览](/tiup/tiup-command-mirror.md)
```

File: TOC.md [hunk 7]
```diff
@@ -183,8 +183,79 @@
...
+          - [tiup mirror modify](/tiup/tiup-command-mirror-modify.md)
+          - [tiup mirror publish](/tiup/tiup-command-mirror-publish.md)
+          - [tiup mirror rotate](/tiup/tiup-command-mirror-rotate.md)
+          - [tiup mirror set](/tiup/tiup-command-mirror-set.md)
+          - [tiup mirror sign](/tiup/tiup-command-mirror-sign.md)
+        - [tiup status](/tiup/tiup-command-status.md)
+        - [tiup telemetry](/tiup/tiup-command-telemetry.md)
+        - [tiup uninstall](/tiup/tiup-command-uninstall.md)
+        - [tiup update](/tiup/tiup-command-update.md)
+      - [TiUP Cluster 命令](/tiup/tiup-component-cluster.md)
```
Comment (qiancai) [inline TOC.md:210]:
```suggestion
      - TiUP Cluster 命令
        - [概览](/tiup/tiup-component-cluster.md)
```

File: TOC.md [hunk 8]
```diff
@@ -183,8 +183,79 @@
...
+        - [tiup cluster rename](/tiup/tiup-component-cluster-rename.md)
+        - [tiup cluster replay](/tiup/tiup-component-cluster-replay.md)
+        - [tiup cluster restart](/tiup/tiup-component-cluster-restart.md)
+        - [tiup cluster scale-in](/tiup/tiup-component-cluster-scale-in.md)
+        - [tiup cluster scale-out](/tiup/tiup-component-cluster-scale-out.md)
+        - [tiup cluster start](/tiup/tiup-component-cluster-start.md)
+        - [tiup cluster stop](/tiup/tiup-component-cluster-stop.md)
+        - [tiup cluster template](/tiup/tiup-component-cluster-template.md)
+        - [tiup cluster upgrade](/tiup/tiup-component-cluster-upgrade.md)
+      - [TiUP DM 命令](/tiup/tiup-component-dm.md)
```
Comment (qiancai) [inline TOC.md:235]:
```suggestion
      - TiUP DM 命令
        - [概览](/tiup/tiup-component-dm.md)
```

File: TOC.md (hunk 2 above)
Comment (qiancai) [reply TOC.md:206]: 同上。概览文件偏场景，命令排序是按字母顺序来的。

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**